analyzer/src/main/resources/init.gradle

```diff
@@ -243,15 +252,15 @@ class AbstractDependencyTreePlugin<T> implements Plugin<T> {
                             dependencyProject.version ?: '', '', '', dependencies, null, '',
                             dependencyProject.projectDir.absolutePath)
                 } else {
-                    return new DependencyImpl('', normalizeName(id.displayName), '', '', '', dependencies,
-                            "Unknown id type: ${id.getClass().simpleName}".toString(), '', null)
+                    return dependencyFromDisplayName(id.displayName, dependencies,
+                            "Unknown id type: ${id.getClass().simpleName}")
```

Comment:
Please keep the `.toString()`, otherwise serialization of the model will fail, same below.

Reply:
I had moved this to inside `dependencyFromDisplayName()`. Do you prefer to keep it at the caller side? My thinking was that it can be easily forgotten there.

Reply:
No, sorry, I missed that.